src/store/drawing/reducer.ts

```diff
@@ -39,7 +47,11 @@ export default createReducer<DrawingState>(initialState, builder =>
                 state.drawnPathGroups = [];
             }
         })
-        .addCase(toggleAnnotationModeAction, () => initialState)
+        .addCase(setDrawingStrokeAction, (state, { payload }) => {
+            const { stroke } = state;
```

Comment:
Could we inline this in L52 since we're only using `stroke` in just one place?

Reply:
Then how to access `state`?